eng/packaging.targets

```
@@ -288,4 +288,19 @@
     <Error Condition="'$(ServicingVersion)' == '0'" Text="ServicingVersion is set to 0 and it should be an increment of the patch version from the last released package." />
   </Target>
 
+  <ItemDefinitionGroup>
+    <TargetPathWithTargetPlatformMoniker>
+      <GeneratePackageOnBuild>$(GeneratePackageOnBuild)</GeneratePackageOnBuild>
+    </TargetPathWithTargetPlatformMoniker>
+  </ItemDefinitionGroup>
+
+  <Target Name="ValidateBuiltPackageInServicing" 
+          Condition="'$(PreReleaseVersionLabel)' == 'servicing' and '$(IsPackable)' == 'true'"
```

This file is only imported when a project sets IsPackable true.

```suggestion
          Condition="'$(PreReleaseVersionLabel)' == 'servicing'"
```